Tools/CISupport/ews-build/steps.py

```diff
@@ -185,6 +185,19 @@ def _is_hash_outdated(self, pr_json):
         return 0 if pr_sha == self.getProperty('github.head.sha', '?') else 1
 
 
+class ShellMixin(object):
+    def shell_command(self, command):
+        platform = self.getProperty('platform', '*')
+        if platform == 'wincairo':
```

Comment:
Should probably be checking whether or not its a windows cmd shell here

Reply:
At the moment, we don't such a property.

What I'm going to do is make the code clear that we're using the platform to decide if we're working with a Windows shell. In the future when/if there are other platforms using a Windows shell, it will be easy to add them or determine if we need a Windows shell in a more centralized place in the build.